releasenotes/notes/add-warning-after-shutdown-tracer-dd8799467c4d29c8.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
```

Comment:
Maybe just use `upgrade` and remind again that this behaviour isn't supported/encouraged?

Feature is a bit weird to me for this change.

Reply:
Yes, this should be an upgrade as it's a breaking change with 1.0.